src/services/peopleSearch.ts

```diff
@@ -74,6 +74,29 @@ export type PeopleSearchQuery = {
   relationship_status?: string;
   involvement: string;
 };
+export interface SelectOption {
+  label: string;
+  value: string;
+}
+
+const getRenamedDepartments = async () => {
```

Comment:
I think we can give this a clearer name, especially since it no longer returns regular strings. Maybe `getDepartmentDropdownOptions`?